setup.py

```diff
@@ -82,10 +81,6 @@
 # modify distutils' behavior.
 cmdclassd = register_commands(PACKAGENAME, VERSION, RELEASE)
 
-# Freeze build information in version.py
-generate_version_py(PACKAGENAME, VERSION, RELEASE,
```

Comment:
@astrofrog or @bsipocz , is it safe to just remove this? Or do I have to replace this with something else?

Reply:
I would use whatever is in the template, both for this and `setup.cfg`
`version = generate_version_py()`